tests/multigrid/transfer_05.with_mpi=true.with_p4est=true.mpirun=3.output

```diff
@@ -0,0 +1,367 @@
+
```

Comment:
Do we really need two variants of this test? I think it would be enough to have this test output and delete the one with `mpirun=1`.